### DISCUSSION TITLE
Different default New Node names

### DISCUSSION THREAD

**Discussion by HubKing** [tsujan/FeatherNotes#359] [Ideas]
I hate those notes apps that force me to specify a name when creating a new page, but FN uses the same "New Node" which makes difficult to know which one is which. I think there could be ways to use different names:

1. Sequential: New Node 1, New Node 2, New Node 3...
2. Date time: `12:34 31/12`
3. Random: Things like `Brave Tiger` or `Tall Lion`. Zellij does this.
![image](https://github.com/tsujan/FeatherNotes/assets/10900489/787cc051-7644-4c5c-adaa-127dd5bd6ec5)

**Comment by tsujan** [comment]
I disagree with all. "New Node" is what it says and can be changed by a simple double-click.
